modules/dnn/src/layers/pooling_layer.cpp

```diff
@@ -654,32 +674,45 @@ class PoolingLayerImpl CV_FINAL : public PoolingLayer
 
     virtual Ptr<BackendNode> initWebnn(const std::vector<Ptr<BackendWrapper> >& inputs, const std::vector<Ptr<BackendNode> >& nodes) CV_OVERRIDE
     {
+        std::cout << "Use WebNN Pooling Layer's Implementation." << std::endl;
         Ptr<WebnnBackendNode> node = nodes[0].dynamicCast<WebnnBackendNode>();
         auto& webnnInpOperand = node->operand;
         auto& webnnGraphBuilder = node->net->builder;
+        Pool2dOptions options;
+        std::vector<int32_t> kernelSize(kernel_size.begin(), kernel_size.end());
+        std::vector<int32_t> Strides(strides.begin(), strides.end());
+        std::vector<int32_t> Padding;
+        if (padMode.empty()) {
+            Padding = {static_cast<int32_t>(pads_begin[0]),
+            static_cast<int32_t>(pads_end[0]),
+            static_cast<int32_t>(pads_begin[1]),
+            static_cast<int32_t>(pads_end[1])};
```

Comment:
The indent of above lines seem not correct. Please check the code format checker in the bot.